packages/javascript/bh-shared-ui/src/components/FinishedJobsTable/FinishedJobsTable.tsx

```diff
@@ -0,0 +1,86 @@
+import { ColumnDef, DataTable, Pagination, Skeleton } from '@bloodhoundenterprise/doodleui';
+import { ScheduledJobDisplay } from 'js-client-library';
+import { FC, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { StatusIndicator } from '../StatusIndicator';
+import { toCollected, toFormatted, toMins, useFinishedJobsQuery } from './finishedJobs';
+
+const COLUMNS_BASE = [
+    {
+        header: () => <div className='pl-4'>ID / Client / Status</div>,
+        id: 'id',
+    },
+    {
+        header: () => <div>Status Message</div>,
+        id: 'status',
+    },
+    {
+        header: () => <div className='min-w-[110px]'>Start Time</div>,
+        id: 'start',
+    },
+    {
+        header: () => <div className='min-w-[85px]'>Duration</div>,
+        id: 'duration',
+    },
+    {
+        header: () => <div>Data Collected</div>,
+        id: 'collected',
+    },
+];
+
+const LOADING_COLS = [
+    () => <Skeleton className='ml-4 h-4' />,
+    () => <Skeleton className='h-4' />,
+    () => <Skeleton className='h-4' />,
+    () => <Skeleton className='h-4' />,
+    () => <Skeleton className='h-4' />,
+];
+
+const FINISHED_JOB_COLS = [
+    ({ row: { original: job } }) => (
```

Comment:
Each item in this array holds the output for a row on the table. The params include `row` which has an `original` prop. This is a single object on the data array that holds the data for that row. Here I am combining destructuring `({ row: { original })` to get the row object directly and I am using `original: job` to alias that object to the name `job`.